applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/model_setup_utilities.cpp

```diff
@@ -89,13 +89,38 @@ ModelPart& CreateModelPartWithASingle3D4NElement(Model& rModel, const Geo::Const
     AddNodalVariablesToModelPart(result, rNodalVariables);
 
     auto nodes =
-        CreateNewNodes(result, {{0.0, 0.0, 0.0}, {1.0, 0.0, 0.0}, {0.0, 1.0, 0.0}, {0.0, 0.0, 1.0}});
+        CreateNewNodes(result, {ElementSetupUtilities::CreatePointsFor3D4NElement()});
     AddDofsToNodes(result.Nodes(), rNodalVariables);
 
-    auto element = make_intrusive<UPwSmallStrainElement<3, 4>>(
-        1, Kratos::make_shared<Tetrahedra3D4<Node>>(nodes), result.CreateNewProperties(0),
-        std::make_unique<ThreeDimensionalStressState>());
+    auto element = ElementSetupUtilities::Create3D4NElement(nodes, result.CreateNewProperties(0));
+    result.AddElement(element);
 
+    return result;
+}
+
+ModelPart& CreateModelPartWithASingle3D8NElement(Model& rModel, const Geo::ConstVariableRefs& rNodalVariables)
+{
+    ModelPart& result = rModel.CreateModelPart("Main");
```

Comment:
I'd suggest to use `auto` here rather than explicitly repeating the type:
```suggestion
    auto& result = rModel.CreateModelPart("Main");
```

This suggestion also applies to lines 117 and 161.

One more thing: to follow the Kratos Style Guide, we may want to change `result` to `r_result`.

Reply:
Done